src/material/schematics/ng-update/migrations/hammer-gestures-v9/import-manager.ts

```diff
@@ -523,3 +528,14 @@ export class ImportManager {
     return commentRanges[commentRanges.length - 1]!.end;
   }
 }
+
+// TODO(crisbeto): backwards-compatibility layer that allows us to support both TS 4.4 and 4.5.
+// Should be removed once we don't have to support 4.4 anymore.
+function createImportSpecifier(
+  propertyName: ts.Identifier | undefined,
```

Comment:
Note: Should probably drop this migration at some point, but this looks good for now 👍